packages/web-components/fast-foundation/src/breadcrumb/breadcrumb.ts

```diff
@@ -0,0 +1,71 @@
+import { FASTElement, observable } from "@microsoft/fast-element";
+import { BreadcrumbItem } from "../breadcrumb-item";
+
+/**
+ * A Breadcrumb Custom HTML Element.
+ *
+ * @public
+ */
+export class Breadcrumb extends FASTElement {
+    /**
+     * @internal
+     */
+    @observable
+    public slottedBreadcrumbItems: HTMLElement[];
+    public slottedBreadcrumbItemsChanged() {
+        if (this.$fastController.isConnected) {
+            if (
+                this.slottedBreadcrumbItems === undefined ||
+                this.slottedBreadcrumbItems.length === 0
+            ) {
+                return;
+            }
+
+            const lastNode: HTMLElement = this.slottedBreadcrumbItems[
+                this.slottedBreadcrumbItems.length - 1
+            ];
+
+            this.removeLastItemSeparator(lastNode);
+            this.setLastItemAriaCurrent(lastNode);
+        }
+    }
+
+    private removeLastItemSeparator(lastNode: HTMLElement): void {
+        if (lastNode instanceof BreadcrumbItem) {
+            (lastNode as BreadcrumbItem).separator = false;
+        }
+    }
+
+    /**
+     * @internal
+     * Finds href on childnodes in the light DOM or shadow DOM.
+     * We look in the shadow DOM because we insert an anchor when breadcrumb-item has an href.
+     */
+    private findChildWithHref(node: HTMLElement): HTMLElement | null {
+        let findNodeWithHref: HTMLElement | null = null;
+
+        if (node.childElementCount > 0) {
```

Comment:
I'm not normally a fan of multiple return statements but in this case, since this is intended to be a small simple function, maybe just return the values inline from the querySelector calls and get rid of the variable findNodeWithHref.